src/atoms/Icon/icons/icn-copy.svg

```diff
@@ -1,6 +1,6 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 18 18">
-    <g fill="none" fill-rule="evenodd" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.455">
-        <path d="M3.91 17H17V3.91H3.91z"/>
-        <path d="M1 14.09V1h13.09"/>
+<svg xmlns="http://www.w3.org/2000/svg" width="18" height="21" viewBox="0 0 18 21">
+    <g fill="none" fill-rule="evenodd" stroke="#163150" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.455">
```

Comment:
This changes the color of all copy icons, I'd prefer to continue using currentColor to have font-driven color choice. If you still need to change the color for Copyable, you can set it in that component's styles.
```suggestion
    <g fill="none" fill-rule="evenodd" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.455">
```